example/src/main/java/com/capgemini/mythaistar/pages/BasePage.java

```diff
@@ -0,0 +1,4 @@
+package com.capgemini.mythaistar.pages;
```

Comment:
Please don't push empty class to develop. Once it has content will be merged. Now please remove this class from pull request